docs/source/concepts/bento.rst

```diff
@@ -1,35 +1,33 @@
-===============
-Building Bentos
```

Comment:
quick ask: Can you also check everywhere else when we reference this file, you might also need to update its link as well.

Reply:
Thanks. The files reference this link directly should be ok as I did not change the file name, but I updated all links that reference sections with the Bentos doc. I also did the same for the Models and Services doc.